tests/src/item-namespace-staking.test.ts

```diff
@@ -0,0 +1,748 @@
+import * as anchor from "@project-serum/anchor";
```

Comment:
I dont love the structure here, its essentially just basic tests for each of the programs but all put into one file now, not to mention the namespace tests are redundant. This is what I propose, do away with this structure, put a file called `item.ts` && `staking.ts` next to the `namespace.ts` file in rust/tests. So we can have a separate file for each programs tests instead of a huge file.